patches/chrome-browser-ui-toolbar-recent_tabs_sub_menu_model_unittest.cc.patch

```diff
@@ -1,12 +1,103 @@
++      CLEAR_BROWSING_DATA_COMMAND_MODEL_DATA
+       {ui::MenuModel::TYPE_SEPARATOR, true},  // <separator>
+       {ui::MenuModel::TYPE_COMMAND, false},   // Recently closed
+       {ui::MenuModel::TYPE_SEPARATOR, true},  // <separator>
+@@ -458,6 +463,7 @@ TEST_F(RecentTabsSubMenuModelTest, OtherDevicesDynamicUpdate) {
+   // Expected menu items with sync enabled:
+   constexpr ModelData kDataSyncEnabled[] = {
+       {ui::MenuModel::TYPE_COMMAND, true},    // History
++      CLEAR_BROWSING_DATA_COMMAND_MODEL_DATA
+       {ui::MenuModel::TYPE_SEPARATOR, true},  // <separator>
+       {ui::MenuModel::TYPE_COMMAND, false},   // Recently closed
+       {ui::MenuModel::TYPE_SEPARATOR, true},  // <separator>
+@@ -477,6 +483,7 @@ TEST_F(RecentTabsSubMenuModelTest, OtherDevicesDynamicUpdate) {
+   // Expected menu items after update:
+   constexpr ModelData kDataAfterUpdate[] = {
+       {ui::MenuModel::TYPE_COMMAND, true},    // History
++      CLEAR_BROWSING_DATA_COMMAND_MODEL_DATA
+       {ui::MenuModel::TYPE_SEPARATOR, true},  // <separator>
+       {ui::MenuModel::TYPE_COMMAND, false},   // Recently closed
+       {ui::MenuModel::TYPE_SEPARATOR, true},  // <separator>
+@@ -504,6 +511,7 @@ TEST_F(RecentTabsSubMenuModelTest, MaxSessionsAndRecency) {
+   // Expected menu items:
+   constexpr ModelData kData[] = {
+       {ui::MenuModel::TYPE_COMMAND, true},    // History
++      CLEAR_BROWSING_DATA_COMMAND_MODEL_DATA
+       {ui::MenuModel::TYPE_SEPARATOR, true},  // <separator>
+       {ui::MenuModel::TYPE_COMMAND, false},   // Recently closed
+       {ui::MenuModel::TYPE_SEPARATOR, true},  // <separator>
+@@ -522,8 +530,8 @@ TEST_F(RecentTabsSubMenuModelTest, MaxSessionsAndRecency) {
+   EXPECT_THAT(base::span<const std::u16string>(
+                   recent_tabs_builder.GetTabTitlesSortedByRecency())
+                   .subspan(0, 3),
+-              ElementsAre(model.GetLabelAt(5), model.GetLabelAt(8),
+-                          model.GetLabelAt(11)));
++              ElementsAre(model.GetLabelAt(6), model.GetLabelAt(9),
++                          model.GetLabelAt(12)));
+ }
+ 
+ TEST_F(RecentTabsSubMenuModelTest, MaxTabsPerSessionAndRecency) {
+@@ -544,6 +552,7 @@ TEST_F(RecentTabsSubMenuModelTest, MaxTabsPerSessionAndRecency) {
+   // Expected menu items:
+   constexpr ModelData kData[] = {
+       {ui::MenuModel::TYPE_COMMAND, true},    // History
++      CLEAR_BROWSING_DATA_COMMAND_MODEL_DATA
+       {ui::MenuModel::TYPE_SEPARATOR, true},  // <separator>
+       {ui::MenuModel::TYPE_COMMAND, false},   // Recently closed
+       {ui::MenuModel::TYPE_SEPARATOR, true},  // <separator>
+@@ -554,12 +563,13 @@ TEST_F(RecentTabsSubMenuModelTest, MaxTabsPerSessionAndRecency) {
        {ui::MenuModel::TYPE_COMMAND, true},    // <tab>
        // max tabs per sessions is 4, so only the 4 most tabs will show,
        // independent of which window they came from.
 +      BRAVE_MAX_TABS_PER_SESSION_AND_RECENCY
```

Comment:
it should be also possible check the test name in `RecentTabsSubMenuModelTest::VerifyModel` and do this change there.